src/components/spinner/spinner.scss

```diff
@@ -45,7 +45,7 @@
 
 :host(.dropdown--spinner) {
     margin: functions.pxToRem(10) 0;
-    color: var(--lime-primary-color, #{mdc-variables.$mdc-theme-primary});
+    // color: var(--lime-primary-color, #{mdc-theme.$primary});
```

Comment:
Just noticed this now. We don't need to fix it here, but this feels like something the consumer should add, rather than the spinner component itself. We only use the `dropdown--spinner` class in `limel-picker`, so this stuff could go in that stylesheet instead of being built into the spinner.